### ISSUE TITLE
Glob support for coverage locations

### ISSUE THREAD

**Issue by hitmands** [paambaati/codeclimate-action#234]
**Is your feature request related to a problem? Please describe.**
It would be great to use `glob` to describe coverage locations

**Describe the solution you'd like**
```
      - name: Jest Testing
        uses: paambaati/codeclimate-action@v2.6.0
        env:
          CC_TEST_REPORTER_ID: ${{ secrets.CODE_CLIMATE_TOKEN }}
        with:
          coverageCommand: yarn test
          coverageLocations: |
            ${{github.workspace}}/packages/*/reports/jest-coverage/lcov.info:lcov
```

**Describe alternatives you've considered**
Listing every package manually

**Comment by Sumolari** [comment]
Is this something you are working on actively?

I'm interesting in this feature and might implement it myself if nobody is currently working on it.

**Comment by paambaati** [comment]
This could be a neat value-add. @Sumolari Thanks, PRs are welcome.

**Comment by paambaati** [comment]
@hitmands Many thanks to @Sumolari, this is now available in [`v2.7.4`](https://github.com/paambaati/codeclimate-action/releases/tag/v2.7.4) 🎉
